PriceCalculatorKata/src/PriceCalculator/Store.cs

```diff
@@ -0,0 +1,49 @@
+namespace PriceCalculator;
+
+public class Store
```

Comment:
interesting creating the concept of a store but it was never mentioned in the requirements.